docs/cloud-native-security/falco-setup.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[ingest-falco]]
+= Ingest Sysdig Falco data
+
+Sysdig Falco (Falco) is an open-source runtime security tool that detects anomalous activity in Linux hosts and containers. You can ingest Falco alerts into {es} to view them on {kib}'s Alerts page and incorporate them into your {elastic-sec} workflows.
+
+First, you'll need to configure {elastic-sec} to receive data from Falco, then you'll need to configure Falco and Falcosidekick to send data to {es}.
+
+[discrete]
+[[ingest-falco-setup-kibana]]
+== Configure {elastic-sec} to receive Falco data
+
+In {elastic-sec}:
+
+. Click **Add integrations**.
+. Search the Integrations page for `Falco`, then select it.
+. Go to the Falco integration's **Settings** tab. 
+. Click **Install Falco**, then confirm by clicking **Install Falco** again. Installation should take less than a minute. 
+
+image::images/falco-kibana-setup.gif[A gif showing the steps to set up Falco in Kibana]
+
+{elastic-sec} is now ready to receive data from Falco. The Falco integration page now has an **Assets** tab where you can inspect the newly installed assets that help to ingest Falco data.
+
+Next, to make alerts from Falco appear on {kib}'s Alerts page:
+
+. Find the **Rules** page in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. Click **Detection rules (SIEM)**.
+. Search for a rule called `External Alerts`. Enable it using the **Enabled** toggle on the right.
+
+image::images/falco-external-alerts-rule.gif[A gif showing the steps to enable the External Alerts rule]
+
+[discrete]
+[[ingest-falco-setup-falco]]
+== Configure Falco and Falcosidekick
```

Comment:
Perhaps we can split this into two sections: the instructions below are for VMs, while for k8s, it's much simpler using Helm chart commands, which you can find in the doc in the initial ticket. I’ve included them below for convenience.
```
# Add falco helm charts
helm repo add falcosecurity https://falcosecurity.github.io/charts
helm repo update

# Install falco and falcosidekick with one chart
helm install falco falcosecurity/falco \
        --set falcosidekick.enabled=true \
        --set tty=true \
        --set driver.kind=modern_ebpf \
        --set collectors.kubernetes.enabled=true \
        --set falcosidekick.config.elasticsearch.hostport="https://<ES host>" \
        --set falcosidekick.config.elasticsearch.username="elastic" \
    --set falcosidekick.config.elasticsearch.password="<password>" \
        --set falcosidekick.config.elasticsearch.index="logs-falco.alerts-all" \
        --set falcosidekick.config.elasticsearch.suffix="none"
```